app/controllers/admin/snap_applications_controller.rb

```diff
@@ -17,12 +17,12 @@ class SnapApplicationsController < Admin::ApplicationController
 
     # See https://administrate-prototype.herokuapp.com/customizing_controller_actions
     # for more information
-    #
     def resend_fax
       application = SnapApplication.find(params[:id])
-      Export.create_and_enqueue!(snap_application: application,
-                                 destination: :fax, force: true)
-      flash[:notice] = "Resent fax for #{application.signature}!"
+      Enqueuer.create_and_enqueue_export!(snap_application: application,
+                                          destination: :fax, force: true)
```

Comment:
Small nit I'm going to pick here.  For consistency and legibility's sake I would break these params out to their own lines. EG:

```ruby
Enqueuer.create_and_enqueue_export!(
  snap_application: application,
  destination: :fax,
  force: true,
)
```